retriever/engines/sqlite.py

```diff
@@ -106,3 +108,27 @@ def get_connection(self):
         import sqlite3 as dbapi
         self.get_input()
         return dbapi.connect(self.opts["file"])
+
+    def fetch_tables(self, dataset, table_name='{db}_{table}', file=os.path.join(DATA_DIR, 'sqlite.db')):
```

Comment:
@adhaamehab, I think we do not need all these parameters. self has everything you would need, including the connection cursor. And we should not select from `sqlite_master`. We should only use the table name function `self.table_name()` to get the name of the table we want.